java/ql/lib/semmle/code/java/security/regexp/NfaUtils.qll

```diff
@@ -890,6 +890,7 @@ module ReDoSPruning<isCandidateSig/2 isCandidate> {
    */
   private predicate isReDoSCandidate(State state, string pump) {
     isCandidate(state, pump) and
+    not epsilonSucc*(state) = AcceptAnySuffix(_) and // pruning early - these can never get stuck in a rejecting state.
```

Comment:
Is there a reason not to use the `acceptsAnySuffix()` helper predicate here and in `reachesOnlyRejectableSuffixes`?

Reply:
No there isn't. I just forgot that I had it.